pages/units/[faction]/[type].tsx

```diff
@@ -69,16 +70,7 @@ const Unit: NextPage<{ unit: UnitDTO; terms: TermsDTO }> = ({
           {unitType.bacterias.map((bacteria) => (
             <tr key={bacteria.bacteriaType}>
               <td>{bacteria.name}</td>
-              <td>
-                <div>{bacteria.description}</div>
-                <div
-                  dangerouslySetInnerHTML={{
-                    __html: bacteria.modifierData
-                      .map((modifier) => modifier.description)
-                      .join("<br />"),
-                  }}
-                />
-              </td>
+              <td dangerouslySetInnerHTML={sanitizeBacteriaData(bacteria)} />
```

Comment:
This will result in a slightly different result, because it's important to always render the description and only `dangerouslySetInnerHTML` of the modifier data.
You can see the difference on `/units/Barya/Hellbreath`.

Previous:
![image](https://user-images.githubusercontent.com/10058950/173034328-ed1944f3-260b-4ff9-beef-60a4f449f4b2.png)

Your change:
![image](https://user-images.githubusercontent.com/10058950/173034268-ad5e918d-062c-4acd-90d9-d8dd05886088.png)

Reply:
Makes sense, good catch. Updating code to adjust for this